contracts/deploy/001_core.js

```diff
@@ -17,7 +17,7 @@ const getStrategyGovernorAddress = async () => {
   const { governorAddr } = await hre.getNamedAccounts();
   if (isMainnet) {
     // On Mainnet the governor is the TimeLock
-    return (await ethers.getContract("MinuteTimelock")).address;
+    return (await ethers.getContract("Timelock")).address;
```

Comment:
Note: It's confusing but currently on Mainnet the governor is the Governor contract.
Now since that is an old migration it's ok to point to the Timelock.